[TM] OUTPOST UPDATE: Новогоднее обновление аванпостов 2026

MrCat15352 opened the pull request:
## Описание / Что этот PR делает
Скоро, да нет, уже снег везде лежит, а скоро ещё и Новый Год! Это время чтобы добавить новогодних ёлок с подарками и обновить интерьер аванпостов!
ЦК получит безлимитную подписку на все подарки. А капитаны, смогут каждый раунд попытать счастье и получить в подарок один раз что-то интересное!

- Обновлен сам зимний аванпост
- - Были обновлены лифты на зимнем аванпосту
- - Были разнесены зоны театра и кухни
- - Объединена и улучшена зона бара с кухней
- - Лифты поменяли свою ориентацию
- - Запитаны автоматы снаружи, оказывается они все 3 года были обесточеные
- Обновлен астероидный аванпост
- Обновлены ангары зимнего аванпоста
- Обновлен ЦК шип
- Добавлены и обновлены AREA
- Добавлен новый товар в раздел civilian & decoration в карго общем. Ящик с гирляндами. Содержит 50 штук гирлянд для украшения стен гирляндами!
- Добавлен снег! Теперь на аванпосту иногда будет идти снег!
- Добавлены гирлянды, снежинки, шарики елочные!
- Добавлены ЁЛКИ С ПОДАРКАМИ! Один раз в раунд на 1 сикей можно получить подарок от святого Войко!
- Добавлены текстуры кирпичной плитки
- Добавлены фонарные столбы под праздник и обычные тоже



## Причина создания ПР / Почему это хорошо для игры
А почему бы и нет? Праздник накануне. Пора включать веселье и улыбку! В прошлом году всем понравились ёлки

## Демонстрация изменений / Тестирование
ЦКК
<img width="829" height="780" alt="image" src="https://github.com/user-attachments/assets/b97c619e-2b32-4dbb-9250-9ff8c2e50436" />

Ледяной аванпост
<img width="685" height="684" alt="image" src="https://github.com/user-attachments/assets/71179f47-0991-4706-991b-5ad2fad0d511" />

Астероидный аванпост
<img width="850" height="728" alt="image" src="https://github.com/user-attachments/assets/9d650653-3035-4500-898b-6f0ec1e2fb79" />

## Список изменений

```yml
🆑
map: добавлено по три ёлки на ЦК, и по одной ёлки на каждый аванпост из двух
/🆑
```

MrCat15352 committed:
Добавляем ёлки!

MrCat15352 committed:
Вернул лото

MrCat15352 committed:
Включаем снег на ледяном аванпосту

MrCat15352 committed:
Новая зона для аванпоста

MrCat15352 committed:
Добавляем зоны и красим аванпост в соответствующие зоны

MrCat15352 committed:
Добавляем музыки для аванпоста

MrCat15352 committed:
переменные и атласы

MrCat15352 committed:
Меняем основы основ ледяного аванпоста

MrCat15352 committed:
Реверт карты

MrCat15352 committed:
Merge remote-tracking branch 'upstream/beta-dev' into new_year_snow_2026

MrCat15352 committed:
Анти реверт реверта поста

MrCat15352 committed:
Вводим всего новогоднего в паки

MrCat15352 committed:
Обновляем и дальше аванпост

MrCat15352 committed:
Merge remote-tracking branch 'upstream/beta-dev' into new_year_snow_2026

MrCat15352 committed:
Merge remote-tracking branch 'upstream/beta-dev' into new_year_snow_2026

MrCat15352 committed:
Завершаем обновление аванпостов + ЦК шип

MrCat15352 committed:
Фиксим и дополняем праздничным настроениме для ящика с гирляндами

MrCat15352 committed:
Задокументировал изменения

MrCat15352 committed:
Добавил новые свитер с оленем в автомат сакуры

Vairkharst reviewed (changes requested):
Разделяй ПР, в таком виде, я не приму. Свалил в кучу непонятно чего, ремап аванпоста, новые вещи, еще непонятно чего.

MrCat15352 committed:
Обновлены фонарные столбы

MrCat15352 committed:
Фиксим ошибку загрузки карт при инициализации. Когда шипы могли быть загружены без владельца

MrCat15352 committed:
Фиксим атмос трубы, рельеф, урезаем зоны, меньше объектов = лучше

MrCat15352 committed:
Исправлено отсутствие падов в посольствах сенткома

MysticalFaceLesS committed:
50 Перенос обновлений с Beta на Alpha сервер (#2575)

## Описание / Что этот PR делает
Я не стану описывать все тут, мне ещё есть чем заняться чем расписывать
чейнж-лог.

## Список изменений

```yml
🆑
add: 3 месяца обновлений
/🆑
```

MrCat15352 committed:
Merge remote-tracking branch 'upstream/alpha-stable' into new_year_snow_2026

MrCat15352 committed:
Удалил лишний товар

MrCat15352 committed:
Вносим фиксы на химку исталин. Вынести потмо в отдельный ПР

MrCat15352 committed:
Фикс исталина

MrCat15352 committed:
Добавлены болты на ангары двух аванпостов, на мед отделение. Все заболтировано

MrCat15352 committed:
Фикс утечки памяти при использовании гирлянды

MrCat15352 committed:
Реверт изменений

MrCat15352 committed:
Заменил механ активных шипов на муляж

MrCat15352 committed:
Удалил полностью спорные шипы

MrCat15352 committed:
Обновлены спрайты столба

MrCat15352 committed:
Поставлены номера лифтов на карту зимнего аванпоста

KOCMODECAHTHUK committed:
Merge branch 'beta-dev' into 2599

KOCMODECAHTHUK committed:
fix: maps

KOCMODECAHTHUK committed:
Merge commit '881417532f76ff2f0ab1fbe1d8f0eee695cf836d' into 2437

KOCMODECAHTHUK committed:
Merge branch 'beta-dev' into new_year_snow_2026
